lib/SimpleSchema.js

```diff
@@ -591,6 +591,32 @@ class SimpleSchema {
     return def.label;
   }
 
+  /**
+   * Gets a field's property
+   *
+   * @param {String} [key] The schema key, specific or generic.
+   *   Omit this argument to get a dictionary of all labels.
+   * @param {String} [prop] Name of the property to get.
+   *
+   * @returns {any} The property value
+   */
+  get(key, prop) {
+    const def = this.getDefinition(key, ['type', prop]);
+
+    if (!def) return undefined;
+
+    if (schemaDefinitionOptions.includes(prop)) {
```

Comment:
Did you mean this to be: 
```javascript
if (_(schemaDefinitionOptions).includes(prop)) {
```
I get a ```TypeError: schemaDefinitionOptions.includes is not a function``` if it is not.  It tests properly for me it this line is changed.